package.json

```diff
@@ -51,10 +51,9 @@
     "@types/compression": "1.7.2",
     "@types/jest": "29.0.3",
     "@types/node": "16.11.62",
-    "@types/react": "17.0.38",
-    "@types/react-dom": "17.0.11",
-    "@types/react-redux": "7.1.21",
```

Comment:
`react-redux` v8 now [comes with built-in typings](https://github.com/reduxjs/react-redux/releases/tag/v8.0.0) so there is no longer a need to use these community created ones.